release-notes/July_2016.md

```diff
@@ -3,6 +3,7 @@ Order: 14
 TOCTitle: July 2016
 PageTitle: Visual Studio Code July 2016 1.4
 MetaDescription: See what is new in the Visual Studio Code July 2016 Release (1.4)
+MetaSocialImage: July_2016_social.jpg
```

Comment:
FYI - @gregvanl notice the pattern in how you can define social images for any page in Docs, Release Notes, or Blog.  You put the directory name of the folder holding the images followed by an underscore and then the images name.  (Same as how the files look on disk after being compiled by gulp)